### PR TITLE
create contests page

### PR DESCRIPTION
**Pull request by sakshi-choudhary**
**ISSUE #224**

**Describe the changes you've made**

Developed Contests Page.

**Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)**
NA

**Checklist**

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

**PREVIEW of work done**

### CODE REVIEW

**Comment by sakshi-choudhary** [comment]
@Abhishek-kumar09 Please do a review.

File: src/data/contestsDetails.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
+[
+  {
+    "ongoing": [
+      {
+        "ContestId": "1",
+        "ContestTitle": "CP Challenge",
+        "Date": "29 August 2021"
```
Comment (Abhishek-kumar09) [inline src/data/contestsDetails.json:7]: please follow camelcasing here.

File: package.json [hunk 2]
```diff
@@ -18,6 +18,7 @@
     "@testing-library/user-event": "^12.1.10",
     "axios": "^0.21.1",
     "capacitor-firebase-auth": "^2.3.5",
+    "chart.js": "^3.0.2",
```
Comment (Abhishek-kumar09) [inline package.json:21]: Have you used this package, anywhere?

**Review by Abhishek-kumar09 — COMMENTED** [review]
Some nitpicks!

Also, don't forget to run `npm run lint:fix` after making your changes.

File: package.json (hunk 2 above)
Comment (sakshi-choudhary) [reply package.json:21]: no, not yet in this PR. I'll remove it.

File: src/data/contestsDetails.json (hunk 1 above)
Comment (sakshi-choudhary) [reply src/data/contestsDetails.json:7]: sure

**Comment by sakshi-choudhary** [comment]
Ready for review, @Abhishek-kumar09

**Review by Abhijay007 — COMMENTED** [review]
@sakshi-choudhary I reviewed your code and there are some changes I want to suggest.

1. You have installed package `chart.js` but haven't used it anywhere. try to remove that if you don't need that package.

2.  the UI looks better if you implement a header here (above contest heading ) with navigation controls.

contest page header

take reference from these two: 

dis-page header
dash page header


3.  Try to wrap text in container format as it looks a bit overflowed in the featured challenges section.

callenges

these are just suggestions 😄 , it is up to you if you want to implement these or not although the code looks good to me.

File: src/pages/Contests/index.js [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+    {
+      status: "completed",
+    },
+  ];
+  const [status, setStatus] = useState("Active");
+  const setStatusFilter = (status) => {
+    setStatus(status);
+  };
+  return (
+    <Container component="main" maxWidth="xs" className={classes.root}>
```
Comment (Abhishek-kumar09) [inline src/pages/Contests/index.js:75]: I think wrapping this container is creating problems with unwanted padding with in appBar

Comment (sakshi-choudhary) [reply src/pages/Contests/index.js:75]: okay, I'll fix it.

**Review by Abhishek-kumar09 — COMMENTED** [review]
run `npm run lint:fix`

**Pull request merged**